x-pack/plugins/apm/server/lib/metrics/get_cpu_chart_data/transformer.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Coordinate } from 'x-pack/plugins/apm/typings/timeseries';
+import { AggValue } from '../query_types';
+import { ESResponse } from './fetcher';
+
+export interface CPUChartAPIResponse {
+  series: {
+    systemCPUAverage: Coordinate[];
+    systemCPUMax: Coordinate[];
+    processCPUAverage: Coordinate[];
+    processCPUMax: Coordinate[];
+  };
+  // overall totals for the whole time range
+  overallValues: {
+    systemCPUAverage: AggValue['value'];
+    systemCPUMax: AggValue['value'];
+    processCPUAverage: AggValue['value'];
+    processCPUMax: AggValue['value'];
```

Comment:
`AggValue['value']` makes it a bit harder to immediately see what's going on where `number | null` is more readable.